testing/load-test/v2/produce-binary-to-topic-test.js

```diff
@@ -0,0 +1,43 @@
+import encoding from "k6/encoding";
+
+import {randomIntBetween, randomItem, uuidv4} from "https://jslib.k6.io/k6-utils/1.0.0/index.js";
+
+import {produceBinaryToTopic, randomByteString} from "./common.js";
+import {createTopic} from "../v3/common.js";
+
+export let options = {
+    stages: [
+        {duration: '10s', target: 2000},
+        {duration: '40s', target: 2000},
```

Comment:
any reason the targets are much higher? what i'm getting from this is that the other tests have REST Proxy as bottle neck. what about in this case?